scripts/release/sync-homebrew-cask.sh

```
@@ -0,0 +1,94 @@
+  APP_BUILD=""
+fi
+
+DMG_PATH="${DMG_PATH:-$REPO_ROOT/build/artifacts/${DMG_BASENAME}.dmg}"
+
+if [[ ! -f "$DMG_PATH" ]]; then
+  echo "DMG not found: $DMG_PATH" >&2
+  echo "Run package-dmg-from-xcarchive.sh first or provide DMG_PATH." >&2
+  exit 1
+fi
+
+if [[ -z "$APP_VERSION" || "$APP_VERSION" == '$('* || -z "$APP_BUILD" || "$APP_BUILD" == '$('* ]]; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)